android/titanium/src/java/org/appcelerator/titanium/view/TiUIView.java

```diff
@@ -1195,7 +1193,7 @@ protected boolean canApplyTouchFeedback(@NonNull KrollDict props)
 	 * @param backgroundColor The background color of the view.
 	 * @param rippleColor The ripple color.
 	 */
-	private void applyTouchFeedback(@NonNull Integer backgroundColor, @Nullable Integer rippleColor)
+	private void applyTouchFeedback(Integer backgroundColor, Drawable backgroundDrawable, @Nullable Integer rippleColor)
```

Comment:
If we're using `@Nullable` to mark rippleColor, we should use the annotations consistently as we go through our code to help us out. So, presumably all three arguments here are `@Nullable` now (since I can see null cheeks below)?

Reply:
Yes, all three parameters can be null. I will add the annotations.